packages/manager/apps/dedicated/client/app/dedicated/server/intervention/template.html

```diff
@@ -0,0 +1,5 @@
+<server-interventions
+    service-name="{{ $ctrl.server.name }}"
+    on-go-back="$ctrl.goBack()"
```

Comment:
I don't see any `onGoBack` binding in component definition ? Should it be added so ? Or removed here ?